compose/docker-compose.acceptance-tests.yml

```diff
@@ -0,0 +1,55 @@
+---
+version: "2"
+
+# Based on "Getting Started with Docker Compose" (https://goo.gl/pBpyyY)
+# More notes here: https://goo.gl/yDLK7f (incl. debugging instructions).
+
+services:
+
+  selenium-hub:
+    image: "selenium/hub:3.6.0"
+    expose:
+      - "4444"
+    ports:
+      - "62100:4444"
+    environment:
+      GRID_BROWSER_TIMEOUT: "10"
+      GRID_DEBUG: "false"
+
+  selenium-firefox:
+    image: "selenium/node-firefox:3.6.0"
+    links:
+      - "selenium-hub"
+      - "nginx"
+    environment:
+      FIREFOX_VERSION: "55.0.3"
+      GECKODRIVER_VERSION: "0.18.0"
+      HUB_PORT_4444_TCP_ADDR: "selenium-hub"
+      HUB_PORT_4444_TCP_PORT: "4444"
+
+  selenium-chrome:
+    image: "selenium/node-chrome:3.6.0"
+    links:
+      - "selenium-hub"
+      - "nginx"
+    volumes:
+      # Mitigates the Chromium issue described at https://goo.gl/fBzwcm
+      - /dev/shm:/dev/shm
+    shm_size: 2g      # TODO: Do I need this?
+    privileged: true  # TODO: Do I need this?
+    environment:
+      CHROME_VERSION: "google-chrome-stable"
+      CHROME_DRIVER_VERSION: "latest"
+      HUB_PORT_4444_TCP_ADDR: "selenium-hub"
+      HUB_PORT_4444_TCP_PORT: "4444"
```

Comment:
I think it defaults to hub (instead of selenium-hub), but I can rename the link below.

Reply:
Like I do in the `archivematica-acceptance-tests` service.